TagManager.php

```diff
@@ -279,7 +280,9 @@ public function regenerateReleasedContainers()
                     }
                 }
             } catch (\Exception $e) {
-                Log::error('There was an error while regenerating container releases: ' . $e->getMessage()  .$e->getTraceAsString());
+                StaticContainer::get(LoggerInterface::class)->error('There was an error while regenerating container releases: {ex}', [
```

Comment:
@diosmosis here as well could we add the trace if the trace is enabled?

Reply:
@tsteur should be done automatically if ExceptionToTextProcessor is enabled

Reply:
Oh, the var name is wrong

Reply:
👍  OK LGTM then. Be great to double check though it works as expected

Reply:
Verified that when the following is in bootstrap.php:

```
define('PIWIK_PRINT_ERROR_BACKTRACE', 1);
```

the stacktrace chain will appear in log output, when, for instance, plugins are deactivated/activated through the UI. This does not happen in CLI mode, however, since Monolog/cli.php overwrites the log handlers.